pkg/migrate/migrate.go

```diff
@@ -124,7 +124,7 @@ func (m *Manager) Run(ctx context.Context, driver Driver, throughRevision string
 
 			log.Info().Str("from", migrationToRun.replaces).Str("to", migrationToRun.version).Msg("migrating")
 
-			in := []reflect.Value{reflect.ValueOf(driver)}
+			in := []reflect.Value{reflect.ValueOf(ctx), reflect.ValueOf(driver)}
```

Comment:
Now that we have generics, we could in theory just make this a generic `func` and remove the use of reflection

Reply:
That's a great idea and the perfect excuse for me to start learning the syntax 😅